src/parachain/collateral.ts

```diff
@@ -13,25 +13,25 @@ export interface CollateralAPI {
      */
     setAccount(account: AddressOrPair): void;
     /**
-     * @returns Total locked DOT collateral
+     * @returns Total locked collateral
      */
-    totalLockedDOT(): Promise<Balance>;
+    totalLocked(): Promise<BN>;
```

Comment:
Thinking about this, would it make more sense to return a Big denominated in DOT actually? We are mostly just handing BTC and DOT amounts in the UI and there is little advantage to have manual denomination conversions